nixos/modules/hardware/ckb.nix

```diff
@@ -10,6 +10,15 @@ in
     options.hardware.ckb = {
       enable = mkEnableOption "the Corsair keyboard/mouse driver";
 
+      gid = mkOption {
+        type = types.nullOr types.str;
```

Comment:
It appears that ckb-daemon only accepts actual GIDs and not group names, so please could we change the type of this option to `types.int` and update the example to reflect this?